.github/ISSUE_TEMPLATE/04_documentation.yml

```diff
@@ -0,0 +1,54 @@
+name: 📚 Documentation issue
+description: Report a possible issue in the documentation
+title: "📚 <TITLE>"
+labels: ["S-To triage", "A-Documentation"]
+body:
+  - type: markdown
+    attributes:
+      value: Thank you for submitting this issue! We'll try to triage it ASAP!
+  - type: markdown
+    attributes:
+      value: |
+        Documentation issue reports that don't follow this template will be closed.
+        Please provide a clear and concise description of what the issue is.
+  - type: input
+    id: url
+    attributes:
+      label: Documentation URL
+      description: Provide the URL of the documentation page where you found the issue.
+      placeholder: https://biomejs.dev/guides/getting-started/
+    validations:
+      required: true
+  - type: dropdown
+    id: language
+    attributes:
+      label: Language
+      description: Language of the documentation page
+      options:
+        - English (en)
+        - Portuguese (pt)
+        - Japanese (ja)
+        - Chinese (zh-CN)
```

Comment:
I was hoping that GitHub was able to provide some options for custom labels, but it doesn't. At this point maybe we should follow your suggestion, and remove the drop-down

Reply:
I could setup a workflow to autotag issues by detecting the language in the URL, if you'd like.

Reply:
That would be awesome!